sources/osgViewer/Viewer.js

```diff
@@ -404,18 +408,12 @@ Viewer.prototype = MACROUTILS.objectInherit( View.prototype, {
         var self = this;
         var render = function () {
             if ( !self.done() ) {
-
-                var isPresentingVR = self._hmd && self._hmd.isPresenting;
-
-                if ( isPresentingVR )
+                if ( self._hmd && self._hmd.isPresenting )
```

Comment:
> ^ that's really nothing, it's not even a function, no dynamic lookup or math, no allocation, just check the visitor/rendering code in osgjs and will see that it's literally nothing in comparison of what is already done.

Yes, I understand your point, clearly we won't experience a better perf. But the render function is somehow a key function, and my point is more from the design, not a perf improvement. 

> Take for example the begin/end Frame that does many many useless tests... https://github.com/cedricpinson/osgjs/blob/develop/sources/osgViewer/Viewer.js#L326-Lundefined

Ouch, this clearly could be rewrited. 

> Anyway I updated the PR, as long as the 2nd parameter doesn't hinder the vr function it's fine by me

Thank you! :+1: